hydrator-common/src/main/java/io/cdap/plugin/common/ConfigUtil.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.common;
+
+import io.cdap.cdap.api.plugin.PluginConfig;
+import io.cdap.cdap.etl.api.FailureCollector;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utilities for plugin configuration
+ */
+public class ConfigUtil {
```

Comment:
why is this needed here?

Reply:
This util was previously used in `google-cloud` repo, but I found it's not `google-cloud` specific and can be exact to even higher level common layer including the new `validateConnection` method I added.

Reply:
see: https://github.com/data-integrations/amazon-s3-plugins/pull/88 and https://github.com/data-integrations/google-cloud/pull/702